packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -15,11 +15,6 @@
 +    # Change this if you change the namespaceOverride or nameOverride of prometheus-operator
 +    url: http://rancher-monitoring-prometheus.cattle-monitoring-system.svc
 +    port: 9090
-+  image:
-+    repository: rancher/mirrored-directxman12-k8s-prometheus-adapter-amd64
-+    tag: v0.8.3
-+    pullPolicy: IfNotPresent
-+    pullSecrets: {}
```

Comment:
removing since this is already tracked in the subchart.

Reply:
will the image still get populated properly for the  image-list? I am not sure how we generate that still, but I remember at one point all the images needed to be listed in the main chart.yaml to ensure it was still picked up for that.

Reply:
I believe @PennyScissors and I had a conversation about this and it does pull through all of the subcharts. It essentially lists every single file within the entire repo, checks whether its name is values.yaml, and lists every single YAML entry within those values.yaml files to find something that has the key image.

Reply:
is this something we should revisit with all of our charts? We patch versions and list them at both levels for all of our charts with subcharts.

Reply:
I think we should. It's a better idea to have them tracked in just one place for reviews and future changes.